.github/workflows/macos_pir_end_to_end_tests.yml

```diff
@@ -8,7 +8,16 @@ on:
   workflow_dispatch:
   schedule:
     - cron: '0 3 * * 1-5' # 3AM UTC offsetted to legacy to avoid action-junit-report@v4 bug
+  push:
+    branches: [ main, "release/**" ]
```

Comment:
Maybe like this?:
```suggestion
    branches: [ main, "release/macos/**", "hotfix/macos/**" ]
```

Reply:
I did wonder about that, but I borrowed this from macos_pr_checks.yml, so we should probably change that one too

Reply:
Is it alright if I do it as part of this PR too? (I'll update the name obviously)